### ISSUE TITLE
Rm issue with the write method

### ISSUE THREAD

**Issue by aureliencarle** [aureliencarle/cryptext#19]

**Comment by GregoireUhlrich** [comment]
For starters, the following 2 lines in the `Touch` command class should be replaced:
https://github.com/aureliencarle/cryptext/blob/a2a3337538a9373997a053191e34d2f73f787ce2/src/commands/touch.py#L17

**Comment by GregoireUhlrich** [comment]
The Touch method should not write down the passwords and read the file again.

It simply should add an item in the `session.content` attribute, after a simple `PasswordData` construction. Something like
```
passwd = PassworDataIO.input()
shell.session.add_password(passwd)
```
The `add_password()` method would have to be defined

**Comment by GregoireUhlrich** [comment]
Finally, the conversion of passwords into the encrypted file should happen when the session closes. 

BTW this should be the responsibility of the `PasswordDataIO` class (move the `convert()` method from `PasswordData` and call it `write_password()` or something like that).

**Comment by GregoireUhlrich** [comment]
After all this is done, the `Rm` class should at least be able to simply remove a password from its key, e.g. 
```
shell.session.remove_password(pass_label)
```
The `remove_password()` method should be implemented

**Comment by aureliencarle** [comment]
High probability that we have a problem with `rm -r machin` cause of the option before parameter ..

**Comment by GregoireUhlrich** [comment]
Indeed. A more sophisticated parser will have to be implemented.. Deserves its own PR probably

**Comment by GregoireUhlrich** [comment]
Implemented successfully in PR #20
